app/javascript/components/ansible-credentials-form/index.jsx

```diff
@@ -13,22 +13,22 @@ const AnsibleCredentialsForm = ({ recordId }) => {
 
   const loadSchema = (value, appendState = {}) => ({
     data: {
-      credential_types: { embedded_ansible_credential_types },
+      credential_types: { EmbeddedAnsibleCredentialTypes },
     },
   }) =>
-    setState(state => ({
+    setState((state) => ({
       ...state,
       ...appendState,
-      fields: embedded_ansible_credential_types[value].attributes,
+      fields: EmbeddedAnsibleCredentialTypes[value].attributes,
     }));
 
   useEffect(() => {
-    API.get('/api/providers?collection_class=ManageIQ::Providers::EmbeddedAutomationManager').then(({ resources: [manager_resource] }) => {
+    API.get('/api/providers?collection_class=ManageIQ::Providers::EmbeddedAutomationManager').then(({ resources: [ManagerResource] }) => {
```

Comment:
Is this form working if manager_resource changed to ManagerResource? i believe api gives manager_resource  data.

Reply:
Yeah I think I initially changed the camel case errors here but forgot to go back and change them back after realizing that the camel case issue can not be changed